anyway/infographics_utils.py

```diff
@@ -1384,9 +1408,11 @@ def get_request_params(
     news_flash_obj: Optional[NewsFlash] = extract_news_flash_obj(news_flash_id)
     if news_flash_obj is None:
         return None
-    location_info = extract_news_flash_location(news_flash_obj)
-    if location_info is None:
+    tmp = extract_news_flash_location(news_flash_obj)
+    if tmp is None:
         return None
+    else:
+        location_info = tmp
```

Comment:
To avoid type warning. location_info is `Dict`, so there is no need to handle the `None` value each time a method receives location_info.

Reply:
After you check `if location_info is None: return ...` the `location_info ` won't be `None` so there is no need to check it after that line also the IDE usually understand that `location_info` won't be `None` so there won't be any warning,
In general at most cases we should not change our code to prevent type warning - we mostly change the type hinting, comments and adding warning suppress .

Reply:
I will rephrase. My goal is: when I get `location_info`, I can trust that it is valid, without verifying there is a check for `None` in the path from the place it was generated to the current location in the code. I think it is worth the addition of these two lines.